src/build.ts

```diff
@@ -0,0 +1,125 @@
+import chalk from 'chalk';
```

Comment:
The source code files in `src/build.ts`, `src/preview.ts` and `src/init.ts` are almost the same as the original  `src/commands/build.ts` `src/commands/preview.ts` `src/commands/init.ts`, but the original commit log has been lost because the file move and the creation of the new file are one commit.
Can you fix it so that the commits for moving the three files and creating the commands endpoint are separated and the commit log is kept?

Reply:
I will check.

Reply:
The history is preserved now. GitHub still renders the diff [the same](https://github.com/vivliostyle/vivliostyle-cli/pull/243/files), unfortunately, but I can verify `git blame` is correct.